From foo@baz Mon Apr  9 10:11:02 CEST 2018
Date: Mon, 09 Apr 2018 10:11:02 +0200
To: Greg KH <gregkh@linuxfoundation.org>
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Subject: net: fec: fix build error in fec driver

From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

commit f4d8124a4ade232cae1161a6aca86e0c0a1fa4f6 which is commit 
a069215cf5985f3aa1bba550264907d6bd05c5f7 upstream caused a build error
in the driver, as the pm functions were not included properly.  So fix
that by including the needed .h file.

Reported-by: Guenter Roeck <linux@roeck-us.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

---
 drivers/net/ethernet/freescale/fec_main.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/freescale/fec_main.c
+++ b/drivers/net/ethernet/freescale/fec_main.c
@@ -58,6 +58,7 @@
 #include <linux/if_vlan.h>
 #include <linux/pinctrl/consumer.h>
 #include <linux/prefetch.h>
+#include <linux/pm_runtime.h>
 
 #include <asm/cacheflush.h>
 
